### PR TITLE
Adding janome

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

_Created with Grayskull_

### CODE REVIEW

**Comment by BastianZim** [comment]
Regarding the tests:

I changed the Grayskull output from 

```yaml
test:
  imports:
    - janome
    - janome.sysdic
  commands:
    - pip check
  requires:
    - pip
```
to 

```yaml
test:
  imports:
    - janome
    - janome.sysdic
```

as specified in the docs. Which one is preferred here?

**Comment by BastianZim** [comment]
Regarding the license:

I added a license as Grayskull wasn't able, but I'm not sure if I did it correctly.

The package specifies a NOTICE.txt that should also be included (See: https://github.com/mocobeta/janome#license). Should I add this as well, or what is the preferred approach?

**Comment by BastianZim** [comment]
Pinging @conda-forge/help-python

Thanks!

**Review by dopplershift — APPROVED** [review]

File: recipes/janome/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+test:
+  imports:
+    - janome
+    - janome.sysdic
+
+about:
+  home: https://github.com/mocobeta/janome
+  summary: Japanese morphological analysis engine.
+  license: Apache-2.0
+  license_file: LICENSE.txt
```
Comment (dopplershift) [inline recipes/janome/meta.yaml:34]: Go ahead and add `NOTICE.txt` and then here you can use a list:
```yaml
license_file:
  - LICENSE.txt
  - NOTICE.txt
```

Comment (BastianZim) [reply recipes/janome/meta.yaml]: Thanks for the quick review. Should be fixed now.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**